backtrace-library/src/main/cpp/backends/backend.cpp

```diff
@@ -82,4 +82,29 @@ void Disable() {
                         "Disable not supported on this backend");
 #endif
 }
+
+jboolean EnableCrashLoopDetectionBackend() {
+#ifdef CRASHPAD_BACKEND
+    return EnableCrashLoopDetectionCrashpad();
+#elif BREAKPAD_BACKEND
+    __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android", "EnableCrashLoopDetection not supported on this backend");
```

Comment:
I think we need to make things more obvious for unity and other tools that will be using it. I think we need to return false here to clearly show it doesn't work